src/lib/components/Swap/Summary/Summary.tsx

```diff
@@ -42,11 +25,15 @@ function TokenValue({ input, usdc, change }: TokenValueProps) {
           {formatCurrencyAmount(input, 6, i18n.locale)} {input.currency.symbol}
         </ThemedText.Body2>
       </Row>
-      {usdc && usdcAmount && (
+      {usdc && (
         <Row justify="flex-start">
           <ThemedText.Caption color="secondary" userSelect>
-            ${formatCurrencyAmount(usdcAmount, 2, i18n.locale)}
-            {change && <Percent gain={change > 0}> {percent}</Percent>}
+            ${formatCurrencyAmount(usdc, 2, i18n.locale)}
+            {priceImpact && (
+              <ThemedText.Caption color={priceImpact?.greaterThan(0) ? 'error' : 'success'}>
```

Comment:
can we use `getPriceImpactWarning` for color here?

Reply:
also documented in [notion](https://www.notion.so/uniswaplabs/0-105-price-impact-is-red-on-swap-review-screen-08a28eef9c6741fdad1cbcecfa976a2a)